### ISSUE TITLE
Compiler error with typescript@4.8.4, sequelize@6.23.2, sequelize-typescript@2.1.3

### ISSUE THREAD

**Issue by spinlud** [sequelize/sequelize-typescript#1441]
Getting compiler error with latest versions of typescript, sequelize and sequelize-typescript:

`typescript`: `4.8.4`
`sequelize`: `6.23.2`
`sequelize-typescript`: `2.1.3`

```
node_modules/sequelize-typescript/dist/model/model/model.d.ts:7:131 - error TS2344: Type 'TModelAttributes' does not satisfy the constraint '{}'.

7 export declare type ModelType<TCreationAttributes, TModelAttributes> = new (values?: TCreationAttributes, options?: any) => Model<TModelAttributes, TCreationAttributes>;
                                                                                                                                    ~~~~~~~~~~~~~~~~

  node_modules/sequelize-typescript/dist/model/model/model.d.ts:7:52
    7 export declare type ModelType<TCreationAttributes, TModelAttributes> = new (values?: TCreationAttributes, options?: any) => Model<TModelAttributes, TCreationAttributes>;
                                                         ~~~~~~~~~~~~~~~~
    This type parameter might need an `extends {}` constraint.

Found 1 error in node_modules/sequelize-typescript/dist/model/model/model.d.ts:7
```

**Comment by BaschtianXD** [comment]
I got the same error. Same versions of the mentioned dependencies.

**Comment by WikiRik** [comment]
The fix will be included in the next release

**Comment by WikiRik** [comment]
Released in 2.1.4
